### ISSUE TITLE
Page Navigation Highlighting is Missing on the 18F Accessibility Guide Writing Guide Page

### ISSUE THREAD

**Issue by jfc3-dol** [18F/accessibility#301]
On the 18F Accessibility Guide Writing Guide page ( https://accessibility.18f.gov/writing-guide/ ) in the left-hand page navigation, for some reason, on this page, the blue left border highlight is missing. Not sure if other navigation highlights are missing, but it's where I noticed it.

**Comment by iamjolly** [comment]
Thanks @jfc3-dol - This should now be fixed via https://github.com/18F/accessibility/pull/304/commits/38e037f23af0bc7fc6e153cf019b26a774845c4f
